lib/galaxy/tools/deps/conda_util.py

```diff
@@ -383,9 +383,9 @@ def install_environment(self):
         a fixed and predictable name given package and version.
         """
         if self.version:
-            return "__%s@%s" % (self.package, self.version)
+            return "__%s_ver_%s" % (self.package, self.version)
         else:
-            return "__%s@_uv_" % (self.package)
+            return "__%s_unver" % (self.package)
```

Comment:
for consistency we'd probably want to make this `return "__%s_unver_" % (self.package)`

Reply:
Why was there a trailing underscore in the first place? I'm being consistent with the versioned name which doesn't end with an underscore ;)